src/Term/AliasGroup.php

```diff
@@ -19,7 +19,7 @@
 class AliasGroup implements Comparable, Countable {
 
 	/**
-	 * @var string
+	 * @var string Requested and actual language of the aliases.
```

Comment:
> Where in this concept is a "request" present?

You are nitpicking about a word. I don't find this helpful. Sure, there are no requests in the DataModel. On the other hand the DataModel is useless without code that uses it. The documentation should say how it's supposed to be used. That's why the variables are called "languageCode", for example, even if there is nothing to ensure it's really a language code.

I could change the comment to something like "usually the requested and actual language code of the aliases".

> what I have a place where I do not want to differentiate between original and fallback language, and want to construct an AliasGroup with a fallback language as language?

If you don't need the additional information you should use the base class and store the fallback language (which is the actual language of the aliases then) in the $languageCode field. Judging from this comment I'm not sure if you fully understood what was going on in #272. That's exactly why I'm trying to make the documentation more clear.